pkg/kubeapiserver/options/authorization.go

```diff
@@ -38,19 +40,57 @@ type BuiltInAuthorizationOptions struct {
 
 func NewBuiltInAuthorizationOptions() *BuiltInAuthorizationOptions {
 	return &BuiltInAuthorizationOptions{
-		Mode: authzmodes.ModeAlwaysAllow,
+		Modes: []string{authzmodes.ModeAlwaysAllow},
```

Comment:
Do we actually want this to be the default?  I don't think so.  Is it even still the default for kube-apiserver?  I thought we switched to RBAC.

Reply:
I think this maybe reserved for test

Reply:
ok